uiowa/uiowa
# Add exposed filters to views block display

Resolves #3804 

# How to test

* Check that this doesn't break v2 person paragraph functionality. Test that it doesn't affect any existing view blocks with exposed filters.

## Stories
- As a non-administrator editing a people block, I am not able to filter by person type status.
- As an administrator, when I select a person type, the person type status options update to reflect on the corresponding options.
